### PR TITLE
added HOST environment

### PR DESCRIPTION
**Pull request by mrpotato42**
I was unsure when modifying env.template

### CODE REVIEW

File: Dockerfile
Comment (HappyMan17) [inline Dockerfile:1]: this variable must be declared in the docker-compose, not in this Dockerfile

File: docker-compose.yml
Comment (HappyMan17) [inline docker-compose.yml:1]: HOST env variable must be set here in the api environtment

**Review by HappyMan17 — CHANGES_REQUESTED** [review]
Remove host from Dockerfile and pass it to docker-compose in api environment variables

File: docker-compose.yml
Comment (mrpotato42) [reply docker-compose.yml:1]: It's not this?

<Event_body_api:
    build:
      context: .
      dockerfile: Dockerfile
    restart: always
    depends_on:
      - db
    environment:
      HOST: ${HOST}
      PORT: ${PORT}
      ...>

this is already done, i think

File: Dockerfile
Comment (mrpotato42) [reply Dockerfile:1]: Complete

**Review by HappyMan17 — COMMENTED** [review]
Well done!

**Pull request merged**